tlsr9/crypto/mbedtls/mbedtls_wrapper.c

```diff
@@ -132,6 +132,38 @@ int __wrap_mbedtls_ecp_muladd(mbedtls_ecp_group *grp,
 	return __wrap_mbedtls_ecp_muladd_restartable(grp, R, m, P, n, Q, NULL);
 }
 
+int __wrap_mbedtls_ecp_gen_keypair_base( mbedtls_ecp_group *grp,
+                     const mbedtls_ecp_point *G,
```

Comment:
Could you remove MBED TLS macro? MBEDTLS_MPI_CHK
Can you follow same code style: no spaces in functions and indentation?